cpu/cpu_freebsd.go

```diff
@@ -133,19 +135,28 @@ func Info() ([]InfoStat, error) {
 	}
 	c.ModelName = strings.Join(vals, " ")
 
-	var cpuNums int32
+	ret := make([]InfoStat, num)
+	for i := 0; i < num; i++ {
+		ret[i] = c
+	}
+
+	return ret, nil
+}
+
+func parseDmesgBoot(fileName string) (InfoStat, int, error) {
+	c := InfoStat{}
+	lines, _ := common.ReadLines(fileName)
+	var cpuNum int
```

Comment:
just pushed b6da2bd76e7d66a228181f7993265cf859d5351f . Thank you for your information!